### PR TITLE
Feature/mysql 8.0

### PR DESCRIPTION
**Pull request by hallelujah**
Trying to validate ability to use mysql 8.0 in circleci

Relates to https://issues.redhat.com/browse/THREESCALE-6825

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM centos/ruby-25-centos7
+FROM quay.io/centos7/ruby-25-centos7
```
Comment (gsaslis) [inline Dockerfile:1]: Should we just go straight to the official image ?

```suggestion
FROM registry.access.redhat.com/ubi8/ruby-25
```

File: Dockerfile [hunk 2]
```diff
@@ -48,7 +48,8 @@ gpgcheck=1\n\
 gpgkey=https://dl-ssl.google.com/linux/linux_signing_key.pub' \
  > /etc/yum.repos.d/google-chrome.repo \
   && yum install -y epel-release \
-  && yum install -y mysql-devel \
+  && yum install -y rh-mysql80-mysql-devel \
+                   rh-mysql80-mysql-config \
```
Comment (gsaslis) [inline Dockerfile:52]: that `rh-` makes me wonder if these packages require a subscription ... 

but I guess we'll find out when we try to build on circleci.

**Comment by gsaslis** [comment]
@hallelujah I wonder if we want the `ruby25` tag to be replaced with the version that has mysql8 vs. whatever-the-previous-mysql-version was... ?

Perhaps we should move this to a new branch `ruby25-mysql8` or something, so we have a different floating tag to use in the container image we then use in the porta CI ?

**Comment by hallelujah** [comment]
I agree about the branch naming, I will create a new branch then

File: Dockerfile (hunk 1 above)
Comment (hallelujah) [reply Dockerfile:1]: So I tried and we can't have the required packages with UBI8 without subscriptions:

- rh-mysql80
- imagemagick 

We could use the community ones but that means there is not much value using UBI8, lets stick to CentOS 7 for now

**Review by gsaslis — APPROVED** [review]
lgtm

let's make sure to also apply the quay.io fix to ruby* branch ;)

**Pull request merged**